spec/controllers/conference_registration_controller_spec.rb

```diff
@@ -1,51 +1,179 @@
 require 'spec_helper'
 
 describe ConferenceRegistrationsController, type: :controller do
-  let(:conference) { create(:conference) }
+  let(:conference) { create(:conference, title: 'My Conference', short_title: 'myconf') }
   let(:user) { create(:user) }
+  let(:not_registered_user) { create(:user) }
+  let(:registered_user) { create(:user) }
+  let!(:registration) { create(:registration, conference: conference, user: registered_user, created_at: 1.day.ago) }
+
+  shared_examples 'access action without ability' do |user, ichain, path, message|
```

Comment:
Yup, the 'without ability' part is misleading. I will change it to 'access #new action'